tests/flows/test_vpr_max_router_iterations.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
You don't need a separate file for each test. We probably need to merge a bunch of these together make it a little cleaner.